packages/prisma/schema.prisma

```diff
@@ -310,7 +310,7 @@ model VerificationToken {
   expiresInDays Int?
   createdAt     DateTime @default(now())
   updatedAt     DateTime @updatedAt
-  teamId        Int?     @unique
+  teamId        Int?
```

Comment:
This was causing that only the most recent invite had the proper team relationship.